genblocks

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
```

Comment:
I would appreciate a comment at the start of this script saying approximately what it does. Makes things so much more easy to follow.